pkg/exec/exec.go

```diff
@@ -37,41 +38,63 @@ var (
 
 func RunCommandInOperatorPod(ctx *k8sutil.Context, cmd string, args []string, operatorNamespace, clusterNamespace string) string {
 
-	pod, err := k8sutil.WaitForOperatorPod(ctx, operatorNamespace)
+	pod, err := k8sutil.WaitForPodToRun(ctx, operatorNamespace, "app=rook-ceph-operator")
 	if err != nil {
 		os.Exit(1)
 	}
 
-	output := new(bytes.Buffer)
+	var stdout, stderr bytes.Buffer
 
-	ExecCmdInPod(ctx, cmd, pod.Name, "rook-ceph-operator", pod.Namespace, clusterNamespace, args, output)
-	return output.String()
+	err = execCmdInPod(ctx, cmd, pod.Name, "rook-ceph-operator", pod.Namespace, clusterNamespace, args, &stdout, &stderr)
+	if err != nil {
+		log.Error(err)
```

Comment:
need to exit after logging the error?

Reply:
actually, no because we want to retry running the command instead returning first time.

Reply:
added